Formula/k/kuzco.rb

```diff
@@ -0,0 +1,45 @@
+class Kuzco < Formula
+  desc "Reviews Terraform and OpenTofu resources and uses AI to suggest improvements"
+  homepage "https://github.com/RoseSecurity/Kuzco"
+  url "https://github.com/RoseSecurity/Kuzco/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "70baccb282e25d74a1d168e3e8e1786431690df9639ca6ef9701f3fdff12a3a4"
+  license "Apache-2.0"
+  head "https://github.com/RoseSecurity/Kuzco.git", branch: "main"
+
+  depends_on "go" => [:build, :test]
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+    generate_completions_from_executable(bin/"kuzco", "completion")
+  end
+
+  test do
+    resource("terraform") do
+      url "https://github.com/hashicorp/terraform/archive/refs/tags/v1.5.7.tar.gz"
+      sha256 "6742fc87cba5e064455393cda12f0e0241c85a7cb2a3558d13289380bb5f26f5"
+    end
+
+    resource("terraform").stage do
+      system "go", "build", *std_go_args(ldflags: "-s -w", output: testpath/"terraform")
+    end
+
+    ENV.prepend_path "PATH", testpath
```

Comment:
why not use opentofu for testing?

Reply:
Although both are supported, the tool defaults to "terraform," so an additional parameter would be required.

Reply:
it would be better to use opentofu, as terraform is deprecated in here :)

Reply:
Even pre-BUSL `1.5.7`? I can make the change, just checking!

Reply:
technically it is fine, but it just makes the test block more cumbersome, and also opentofu would diverge from terraform eventually.

Reply:
Just added opentofu! Thanks for the suggestion